src/web/app/pages-instructor/instructor-session-result-page/instructor-session-result-question-view.component.html

```diff
@@ -1,6 +1,6 @@
 <div class="card margin-bottom-15px" *ngFor="let question of questionsOrder">
   <div class="card-header bg-info text-white course-tab-header" (click)="toggleAndLoadTab.emit(question.question.feedbackQuestionId)">
-    <tm-question-text-with-info [questionNumber]="question.question.questionNumber" [questionDetails]="question.question.questionDetails" (click)="$event.stopPropagation()"></tm-question-text-with-info>
+    <tm-question-text-with-info [questionNumber]="question.question.questionNumber" [questionDetails]="question.question.questionDetails" (click)="toggleAndLoadTab.emit()"></tm-question-text-with-info>
```

Comment:
Try to understand what the original `$event.stopPropagation()` is doing. Hint: Click on the `more` button. It is not supposed to expand the tab.

Also, if you open the console, you will find a lot of errors when you expand the tab, due to this line change.